tests/blas/nodes/axpy_test.py

```diff
@@ -140,12 +146,13 @@ def pure_graph(veclen, precision, implementation="pure", testCase="0"):
                                memlet=Memlet.simple(z_out,
                                                     "0:n/{}".format(veclen)))
 
-    test_sdfg.expand_library_nodes()
+    # test_sdfg.expand_library_nodes()
+    saxpy_node.expand(test_sdfg, test_state, vec_width=veclen)
 
     return test_sdfg.compile()
 
 
-def test_pure():
+def _test_pure():
```

Comment:
that would probably be fine to keep as test_pure() so that our CPU tests run it automatically